### DISCUSSION TITLE
About the FAQ Question "Why was the text entry feature removed?"... Isn't there a text entry feature?

### DISCUSSION THREAD

**Discussion by Justformas** [Monika-After-Story/MonikaModDev#9086] [QA - tech support + general] [answered]
Unless this is referring to some other text entry feature that was in the game, there is a text entry feature in the game isn't there? Or maybe it was re-added. The text entry feature I'm referring to is being able to type out letters and words near the top right of the screen, to search for topics. That feature can dead end like what the FAQ was talking about, but I still think it's very useful and don't want it removed, especially with how many topics there are.

So what is that question in the FAQ referring to? Or is it outdated and no longer relevant?

**Comment by multimokia** [comment, marked as answer]
> The text entry feature I'm referring to is being able to type out letters and words near the top right of the screen, to search for topics

The feature you're describing is the **search bar**, not the text entry feature.

The one described on the FAQ is an old system which allowed you to type directly to Monika instead of going through either the `Hey, Monika...` or `Repeat Conversation` menu options (which didn't exist at the time)

**Reply by Justformas** [reply]
Ah, so the menu options and search bar basically replaced the text entry feature. That's surprising to me that those menu options didn't even exist at one point. Between not knowing that stuff, and the search bar still involving "entering text", that's where the confusion stemmed from.

The search bar can still dead end too of course, but that doesn't detract from its value to me. Even that can be useful to know that there isn't a topic with that text, or that you haven't unlocked it yet.

Thank you for the concise answer.
